client/post-editor/media-modal/gallery/fields.jsx

```diff
@@ -95,6 +98,10 @@ export default React.createClass( {
 		this.props.onUpdateSetting( 'orderBy', event.target.checked ? 'rand' : null );
 	},
 
+	updateReverseOrder() {
+		this.props.onReverse();
```

Comment:
I'd suggest either:

- Checking that the `onReverse` prop exists before calling on it
- Providing a `noop` (either `() => {}` or [Lodash's `_.noop`](https://lodash.com/docs/4.16.4#noop) as a default prop
- Marking the prop as `.isRequired` in `propTypes`

My preference is toward the second